pkgs/by-name/js/jsonschema/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "jsonschema";
+  version = "5.3.1";
+
+  src = fetchFromGitHub {
+    owner = "santhosh-tekuri";
+    repo = "jsonschema";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-ANo9OkdNVCjV5uEqr9lNNbStquNb/3oxuTfMqE2nUzo=";
+  } + "/cmd/jv";
+
+  vendorHash = "sha256-FuUkC7iwn/jO3fHjT9nGUXc2X1QuuxPc8DAzVpzhANk=";
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  meta = with lib; {
+    description = "json-schema compilation and validation";
```

```suggestion
    description = "JSON schema compilation and validation";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes
* `meta.description` must: 
  * Be capitalized.